# The same as https://legacy.uavcan.org/Specification/7._List_of_standard_data_types/#rawairdata
Header header
float32 static_pressure             # Pascal
float32 differential_pressure       # Pascal
